pandas/io/spss.py

```diff
@@ -0,0 +1,27 @@
+def read_spss(path, usecols=None, categorical=True):
```

Comment:
can you add typing

Reply:
Sure. This is the first time I've used typing so please double-check.